src/mudlet.cpp

```diff
@@ -2084,15 +2084,16 @@ bool mudlet::setLabelOnLeave(Host* pHost, const QString& name, const QString& fu
     }
 }
 
-int mudlet::getLineNumber(Host* pHost, QString& name)
+std::pair<bool, int> mudlet::getLineNumber(Host* pHost, QString& windowName)
 {
     QMap<QString, TConsole*>& dockWindowConsoleMap = mHostConsoleMap[pHost];
-    if (dockWindowConsoleMap.contains(name)) {
-        return dockWindowConsoleMap[name]->getLineNumber();
+    const auto window = dockWindowConsoleMap.constFind(windowName);
+    if (window != dockWindowConsoleMap.cend()) {
+        return make_pair(true, window.value()->getLineNumber());
     } else {
-        TDebug(QColor(Qt::white), QColor(Qt::red)) << "ERROR: window doesn't exist\n" >> 0;
+        TDebug(QColor(Qt::white), QColor(Qt::red)) << QStringLiteral("ERROR: window doesn't exist\n") >> 0;
```

Comment:
Have we decided that the **Central Debug Console** (which is where `TDebug` output goes isn't is?) should NOT be subject to translation? I had the impression that it was something that we would be doing when we got around to it...